lib/circuit.js

```diff
@@ -93,6 +94,9 @@ class CircuitBreaker extends EventEmitter {
     if (this.options.cache) {
       CACHE.set(this, undefined);
     }
+
+    // Register this instance of the circuit breaker with the hystrix stats listener
+    hystrixStats.register(this[STATUS]);
```

Comment:
With my [latest push](https://github.com/bucharest-gold/opossum/commit/7bb3931c9df3df80cde3cb54cdae5cf6ef18b4e0), this should be `this[STATUS].stats`. That will return an `Object` that has the properties you can see [here](https://github.com/bucharest-gold/opossum/blob/master/lib/status.js#L117-L126). It represents the cumulative data for each event over the entire window.